pinot-segment-local/src/test/resources/data/newColumnsSchemaWithForwardIndexDisabled.json

```diff
@@ -0,0 +1,77 @@
+{
+  "schemaName": "testDataMV",
+  "dimensionFieldSpecs": [
+    {
+      "name": "column1",
+      "dataType": "INT"
+    },
+    {
+      "name": "column2",
+      "dataType": "INT"
+    },
+    {
+      "name": "column3",
+      "dataType": "STRING"
+    },
+    {
+      "name": "column4",
+      "dataType": "STRING"
+    },
+    {
+      "name": "column5",
+      "dataType": "STRING"
+    },
+    {
+      "name": "newForwardIndexDisabledColumnSV",
```

Comment:
Do we have test for this feature with one of INT / FLOAT / LONG ... other than STRING ?

Reply:
Do you want me to add a test in these files for other types? My other query tests use other types. Is that sufficient?

Reply:
As long as we have coverage, I am fine.

Reply:
cool then we should be good. as part of reload i can perhaps test out more types on reload path too